[2025-08-17T05:05:54.903+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: churn_pipeline_dag.data_validation manual__2025-08-17T05:05:48.819770+00:00 [queued]>
[2025-08-17T05:05:54.911+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: churn_pipeline_dag.data_validation manual__2025-08-17T05:05:48.819770+00:00 [queued]>
[2025-08-17T05:05:54.912+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-08-17T05:05:54.924+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): data_validation> on 2025-08-17 05:05:48.819770+00:00
[2025-08-17T05:05:54.932+0000] {standard_task_runner.py:57} INFO - Started process 89 to run task
[2025-08-17T05:05:54.936+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'churn_pipeline_dag', 'data_validation', 'manual__2025-08-17T05:05:48.819770+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/churn_pipeline_dag.py', '--cfg-path', '/tmp/tmpvrj_715x']
[2025-08-17T05:05:54.937+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask data_validation
[2025-08-17T05:05:54.995+0000] {task_command.py:415} INFO - Running <TaskInstance: churn_pipeline_dag.data_validation manual__2025-08-17T05:05:48.819770+00:00 [running]> on host d1471a41dc29
[2025-08-17T05:05:55.073+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='churn_pipeline_dag' AIRFLOW_CTX_TASK_ID='data_validation' AIRFLOW_CTX_EXECUTION_DATE='2025-08-17T05:05:48.819770+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-17T05:05:48.819770+00:00'
[2025-08-17T05:05:55.957+0000] {python.py:194} INFO - Done. Returned value was: Starting data validation...
data_lake/raw/source=web_logs/ingestion_date=2025-08-17/run_id=20250817_050552/part-00001.csv passed all checks
Data quality report saved to reports/data_quality_report.csv

[2025-08-17T05:05:55.989+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=churn_pipeline_dag, task_id=data_validation, execution_date=20250817T050548, start_date=20250817T050554, end_date=20250817T050555
[2025-08-17T05:05:56.032+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-17T05:05:56.057+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
